tests/framework/pod.go

```diff
@@ -0,0 +1,140 @@
+package framework
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/utils/pointer"
+
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	busyboxImage = "quay.io/quay/busybox:latest"
+)
+
+func PodWithPvcSpec(podName, pvcName string, cmd, args []string) *v1.Pod {
+	volumeName := "pv1"
+	// uid is used as the qemu group in fsGroup
+	const uid int64 = 107
+
+	return &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: podName,
+			Annotations: map[string]string{
+				"cdi.kubevirt.io/testing": podName,
+			},
+		},
+		Spec: v1.PodSpec{
+			SecurityContext: &v1.PodSecurityContext{
+				FSGroup: pointer.Int64(uid),
```

Comment:
I think the `pointer` library is deprecated and you should use the `ptr` library `k8s.io/utils/ptr`. This becomes something like this
```go
ptr.To[int64](uid)
```

Reply:
done